pkg/sdn/api/types.go

```diff
@@ -44,7 +44,7 @@ type NetNamespace struct {
 	kapi.ObjectMeta
 
 	NetName string
-	NetID   uint
+	NetID   *uint
```

Comment:
It's persisted as JSON, so it will always have a number currently. *uint just opens the possibility for `null` as additional value. The difference would be if someone didn't specify `netid` before, it would come in as `0`. Do we want to preserve that behavior or make it so that not specifying a `netid` comes in as `nil`?